acc_test/data/valid_with_vpc_peering.tf

```diff
@@ -71,7 +71,7 @@ resource "instaclustr_cluster" "gcp_example" {
 
   bundle {
     bundle = "APACHE_CASSANDRA"
-    version = "3.11.8"
```

Comment:
These older versions were applied as bundle overrides to the test account, so if we are updating the version, be sure to apply overrides for the new version too so we aren't constantly updating the bundle versions as we roll out new versions

Reply:
I changed this because I saw someone changed the version of a “find and replace” command in some tests probably without realising what they changed? So, basically these versions changed in the data folder are for matching the “find” part of the “find and replace” command otherwise whatever people wants to replace into those templates in the data folder would not be affected because the specified version in that command is never found. That’s my understanding though which could be wrong. Please lemme know if that’s the case and I’ll push a patch to them.

Reply:
The "find and replace" feature, which comes from `AccClusterResourceTestSteps` which isn't used by most of these changed files from the looks of it, it is used in `valid.tf` though so that certainly needs to be updated - thanks for that. 

For these other files, it's fine to use the newer version as long as the tests are passing, and we have a bundle version override for it on the test account so that these tests don't need to be updated when we release a new bundle version.

Reply:
Oh great that's good to know. I do realise that i have commented out a few other tests that were using some templates in the "data" folder and there's no point making those changes anymore. But I also remember before commenting them out, they were failing on the replace command so that's why i made those changes in the first place.